docs/source/resources/package-spec.rst

```diff
@@ -284,86 +283,98 @@ parts:
 * The first part is always the exact name of the package.
 
 * The second part refers to the version and may contain special
-  characters:
+  characters.  See table below.
 
-  * \| means OR.
+* The third part is always the exact build string. When there are
+  3 parts, the second part must be the exact version.
 
-    EXAMPLE: ``1.0|1.2`` matches version 1.0 or 1.2.
+.. list-table:: Version Special Characters
+   :widths: 10, 40, 40
+   :header-rows: 1
 
-  * \* matches 0 or more characters in the version string. In
-    terms of regular expressions, it is the same as ``r'.*'``.
+   * - Symbol
+     - Meaning
+     - Example
 
-    EXAMPLE: 1.0|1.4* matches 1.0, 1.4 and 1.4.1b2, but not 1.2.
+   * - <, >, <=, >=
+     - Relational operators on versions, |br| which are compared using `PEP-440 <https://www.python.org/dev/peps/pep-0440/>`_.
+     - ``<=1.0`` matches 0.9, 0.9.1, and 1.0, but not 1.0.1.
 
-  * <, >, <=, >=, ==, and != are relational operators on versions,
-    which are compared using
-    `PEP-440 <https://www.python.org/dev/peps/pep-0440/>`_. For example,
-    ``<=1.0`` matches ``0.9``, ``0.9.1``, and ``1.0``, but not ``1.0.1``.
-    ``==`` and ``!=`` are exact equality.
+   * - ==, and !=
+     - Exact equality and not equalities.
+     - ``==0.5.1`` matches 0.5.1 and not anything else while ``!=0.5.1`` matches everything but.
 
-    Pre-release versioning is also supported such that ``>1.0b4`` will match
-    ``1.0b5`` and ``1.0rc1`` but not ``1.0b4`` or ``1.0a5``.
+   * - ~=
+     - Compatibility Release
+     - ``~=0.5.3`` equivalent to say ``>=0.5.3, <0.6.0a``
```

Comment:
```suggestion
     - ``~=0.5.3`` is equivalent to ``>=0.5.3, <0.6.0a``
```

Reply:
I'm not fully understanding what this means, though. Are there packages in between 0.5.3 and 0.6.0a in this example?

Reply:
There could be.  The compatibility release operator is designed to future proof a project under the hope that libraries follow semantic version promises.